api/rosetta/errors.go

```diff
@@ -35,13 +35,15 @@ const (
 	txLength         = "transaction identifier has invalid hash filed length"
 	txInvalidOpCount = "transaction operations list has an invalid number of operations"
 
-	networkCheck      = "unable to check network"
-	blockRetrieval    = "unable to retrieve block"
-	balancesRetrieval = "unable to retrieve balances"
-	oldestRetrieval   = "unable to retrieve oldest block"
-	currentRetrieval  = "unable to retrieve current block"
-	txRetrieval       = "unable to retrieve transaction"
-	intentDetermine   = "unable to determine transaction intent"
+	networkCheck              = "unable to check network"
+	blockRetrieval            = "unable to retrieve block"
+	balancesRetrieval         = "unable to retrieve balances"
+	oldestRetrieval           = "unable to retrieve oldest block"
+	currentRetrieval          = "unable to retrieve current block"
+	txRetrieval               = "unable to retrieve transaction"
+	intentDetermine           = "unable to determine transaction intent"
+	txReferenceRetrieval      = "unable to retrieve transaction reference block"
+	txSequenceNumberRetrieval = "unable to retrieve account key sequence number"
```

Comment:
`intentDetermine` doesn't make sense as shortened error here. It should be `invalidIntent` or `unknownIntent`, or even `undefinedIntent`, depending on the semantics.

For the second, it can be `referenceBlockRetrieval`, but why not just use `blockRetrieval` here, is there a difference?

For the third, it can be `sequenceNumberRetrieval`.